doc/overview.rst

```diff
@@ -830,8 +813,78 @@ tune the model for your use case with your custom dataset. The following assumes
--------------------------------------------------
+Low-code fine-tuning with the JumpStartEstimator class
+------------------------------------------------------
+
+You can fine-tune a built-in algorithm or pre-trained model in just a few lines of code using the SageMaker Python SDK.
+
+First, find the model ID for the model of your choice in the :doc:`Built-in Algorithms with pre-trained Model Table<./doc_utils/pretrainedmodels>`.
+
+Using the model ID, define your training job as a JumpStart estimator. Run ``estimator.fit()`` on your model, pointing to the training data to use for fine-tuning.
+Then, use the ``deploy`` method to automatically deploy your model for inference. In this example, we use the GPT-J 6B model from HuggingFace.
+
+.. code:: python
+
+    from sagemaker.jumpstart.estimator import JumpStartEstimator
+
+    model_id = "huggingface-textgeneration1-gpt-j-6b"
+    estimator = JumpStartEstimator(model_id=model_id)
+    estimator.fit(
+        {"train": training_dataset_s3_path, "validation": validation_dataset_s3_path}, logs=True
+    )
+    predictor = estimator.deploy()
+
+You can then run inference with the deployed model using the ``predict`` method.
+
+.. code:: python
+
+    question = "What is Southern California often abbreviated as?"
+    response = predictor.predict(question)
+    print(response)
+
+You can optionally include specific model versions or instance types. For more information about the ``JumpStartEstimator`` class and its parameters,
+see `JumpStartEstimator <https://sagemaker.readthedocs.io/en/stable/api/inference/model.html#sagemaker.jumpstart.estimator.JumpStartEstimator>`__.
+
+Additional low-code training utilities
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can optionally include specific model versions or instance types when fine-tuning a pretrained model
+using the ``JumpStartEstimator`` class. All JumpStart models have a default instance type.
+Retrieve the default training instance type using the following code:
+
+.. code:: python
+
+    from sagemaker import instance_types
+
+    instance_type = instance_types.retrieve_default(
+        model_id=model_id,
+        model_version=model_version,
+        scope="training")
+    print(instance_type)
+
+See all supported instance types for a given JumpStart model with the ``instance_types.retrieve()`` method.
+
+To check the default hyperparameters used for training, you can use the ``retrieve_default()`` method
+from the ``hyperparameters`` class.
+
+.. code:: python
+
+    from sagemaker import hyperparameters
+
+    hyperparameters = hyperparameters.retrieve_default(model_id=model_id, model_version=model_version)
+    print(hyperparameters)
+
+    # Optionally override default hyperparameters for fine-tuning
+    hyperparameters["epoch"] = "3"
+    hyperparameters["per_device_train_batch_size"] = "4"
+
+You can also check the default metrics definitions:
+.. code:: python
+
+    print(estimator.metric_definitions)
```

Comment:
could you demonstrate using the `metric_definitions.retrieve_default` [utility](https://github.com/aws/sagemaker-python-sdk/blob/6a252f9ab16febdddb0c7b691677e6c0db41f826/src/sagemaker/metric_definitions.py#L26) please?

Reply:
When I tested this, using metric_definitions.retrieve_default() resulted in an AttributeError: 'list' object has no attribute 'retrieve_default'. Using print(estimator.metric_definitions) got the following response: [{'Name': 'hugginface-textgeneration:eval-loss', 'Regex': "'eval_loss': ([0-9]+\\.[0-9]+)"}, {'Name': 'hugginface-textgeneration:train-loss', 'Regex': "'loss': ([0-9]+\\.[0-9]+)"}].

Reply:
This is surprising, I'll try tomorrow, but the python code should be:
```py
from sagemaker import metric_definitions

my_metric_definitions = metric_definitions.retrieve_default(...)
```

Reply:
In any case, that's non-blocking, we can send another PR later to demonstrate how to use this method.